gdb-xml/mips64-cheri-c128.xml

```diff
@@ -70,8 +70,8 @@
 
   <reg name="ddc" bitsize="128" type="cheri_cap128"/>
   <reg name="pcc" bitsize="128" type="cheri_cap128"/>
-  <reg name="UserTls" bitsize="128" type="cheri_cap128"/>
```

Comment:
The renames are fine.  I tried to name them based on the variables we used in QEMU.  I think the renames don't matter if you try just the QEMU changes without changing GDB.